src/api/providers/openrouter.ts

```diff
@@ -151,6 +151,7 @@ export class OpenRouterHandler implements ApiHandler {
 			const response = await axios.get(`https://openrouter.ai/api/v1/generation?id=${genId}`, {
 				headers: {
 					Authorization: `Bearer ${this.options.openRouterApiKey}`,
+					"User-Agent": "Cline",
```

Comment:
Might want to include the version info like this PR: https://github.com/cline/cline/pull/1485

Reply:
sorry, my fault, I actually talked to @yrobla about taking over/merging her PR.

Reply:
@yrobla I can't push to your fork. Could you cherry-pick https://github.com/cline/cline/commit/442bd41dbcc3a823bccd45fc92fc9e22b77395a1 and re-push? Alternatively I can include those patches in #1485